Amcovad/amcovad-frontend
# #180822215 Create an Input Component

#### What does this PR do?
`Create a reuseable Input Component`
#### Description of Task to be completed?
- An Input component that can be passed to several other components
- Added and `SVG Icon` props to make the input reusable with or without an Icon
#### How should this be manually tested?
- Clone the Repository
- Run `yarn Install` in your terminal to install required dependencies
-  Run `yarn dev` to start the local development server 
- Open your browser and check `localhost:3000`
